apps/website/src/components/content/YouTube.tsx

```diff
@@ -56,19 +58,35 @@ type TriggerProps = {
 };
 
 const createTrigger = (id: string) => {
-  const Trigger = ({ videoId, caption, className, children }: TriggerProps) => {
-    return (
-      <a
-        href={`https://www.youtube.com/watch?v=${encodeURIComponent(videoId)}`}
-        target="_blank"
-        rel="noreferrer"
-        className={classes("group/trigger", className)}
-        {...{ [`data-lightbox-${id}`]: JSON.stringify({ videoId, caption }) }}
-      >
-        {children}
-      </a>
-    );
-  };
+  const Trigger = forwardRef<() => void, TriggerProps>(
+    ({ videoId, caption, className, children }, ref) => {
+      // Expose a method to open the lightbox as the ref
+      const elm = useRef<HTMLAnchorElement>(null);
+      const open = useCallback(() => {
+        if (elm.current) elm.current.dispatchEvent(new Event("lightbox"));
+      }, []);
+      useEffect(() => {
+        if (typeof ref === "function") ref(open);
+        else if (ref) ref.current = open;
+      }, [ref, open]);
```

Comment:
My only question is whether it makes sense to expose a custom method here, or whether I should just expose the element and let the consumer dispatch the event?